homeassistant/components/zwave_mqtt/config_flow.py

```diff
@@ -0,0 +1,24 @@
+"""Config flow for zwave_mqtt integration."""
+from homeassistant import config_entries
+
+from .const import DOMAIN  # pylint:disable=unused-import
+
+TITLE = "Z-Wave MQTT"
+
+
+class DomainConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for zwave_mqtt."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_PUSH
+
+    async def async_step_user(self, user_input=None):
```

Comment:
For future, could we do a subscribe to `OpenZWave/+/status` and wait a second for any retained messages. This way we can check if there is any instance online and not finish set up if we can't find any.

Reply:
I've made a note of this.